db.go

```diff
@@ -82,6 +82,8 @@ type UserStore interface {
 	// TODO RenameIdentity was deprecated in May 2016, replaced by UpdateIdentity. We need to remove this once PCS team has made the necessary updates
 	RenameIdentity(oldIdent, newIdent string) error                        // Rename an identity. Returns ErrIdentityAuthNotFound if oldIdent does not exist. Returns ErrIdentityExists if newIdent already exists.
 	GetUserFromIdentity(identity string) (AuthUser, error)                 // Gets the user object from the identity supplied
+	LockAccount(userId UserId) error                                       // Locks account after a certain number failed log in attempts, determined by MaxFailedLoginAttempts in config
```

Comment:
Comment should probably change - especially if we make the LockingPolicy more flexible.
It's not necessarily after X number of failed login attempts. It could be for any reason.